app/controllers/cases/case/subcases/add-subcase.js

```diff
@@ -7,6 +7,6 @@ export default class CasesCaseSubcasesAddSubcaseController extends Controller {
 
   @action
   onCreateSubcase() {
-    this.router.refresh('cases.case.subcases');
+    this.router.refresh('cases.case.index');
```

Comment:
This is called after a `subcase` is created. In theory this would mean it will always route to the latest subcase, which would be fine. But we do need to verify this during testing. If not, this needs to stay on the `subcases` route.

Reply:
Will revert to subcase route